crates/ruff_python_ast/src/nodes.rs

```diff
@@ -1654,6 +1654,14 @@ impl StringLiteralValue {
         self.iter().fold(0, |acc, part| acc + part.value.len())
     }
 
+    /// Returns the number of string literals in this value.
+    pub fn count(&self) -> usize {
```

Comment:
Nit: I wouldn't call this `count` because it isn't actually counting. It's also unclear how it is different from `len` (which probably should return a `TextSize`)

Maybe `literals_len`. The alternative is to add `len` as a method to `Iter` and add a `literals` method so that it reads as `string.literals().len()`

Reply:
Why do you think it isn't counting? I can change it to `.as_slice().len()` or `.iter().count()` - both are available.

Reply:
I've changed it to `.as_slice().len()`